### DISCUSSION TITLE
Nuxt SSR docs are 404ing

### DISCUSSION THREAD

**Discussion by ChrisRenno** [vercel/community#2912] [Bug Report]
### Page to Investigate

https://vercel.com/docs/concepts/docs/frameworks/nuxt#server-side-rendering-ssr

### Steps to Reproduce

- Go to https://vercel.com/docs/concepts/incremental-static-regeneration
- Find and click the link - See [our dedicated Nuxt docs](https://vercel.com/docs/concepts/docs/frameworks/nuxt#server-side-rendering-ssr) to use ISR with Nuxt

**Comment by leerob** [comment]
Thanks, we'll get this fixed! It should be https://vercel.com/docs/frameworks/nuxt#server-side-rendering-ssr
